dashboard/public/config.json

```diff
@@ -1,4 +1,5 @@
 {
   "namespace": "default",
-  "appVersion": "DEVEL"
+  "appVersion": "DEVEL",
+  "logoutURI": "/oauth2/sign_out"
```

Comment:
I am not sure why the logoutURI is configurable? Isn't that related to oauth2_proxy?

Reply:
Yes, but using oauth2_proxy on the kubeapps frontend is just the option we support in the chart - you could also deploy your own oauth2 proxy or whatever on a different host and need a full URI here. That was my thinking anyway, happy to remove it if you think so.

Reply:
I see, that's fine (I was unsure if that was related to the IdP). We don't support anything else apart than oauth2_proxy but I am fine leaving that.